takhojin/app.js

```diff
@@ -0,0 +1,40 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+
+dotenv.config();
+
+const myDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized");
+});
```

Comment:
```suggestion
const dataSource = new DataSource({
  type: process.env.DB_CONNECTION,
  host: process.env.DB_HOST,
  port: process.env.DB_PORT,
  username: process.env.DB_USERNAME,
  password: process.env.DB_PASSWORD,
  database: process.env.DB_DATABASE,
});

dataSource.initialize().then(() => {
  console.log("Data Source has been initialized");
});
```
1. myDataSource 보다는 `appDataSource` , `mysqlDataSource` 혹은 간결하게 `database`로 변수명을 변경해주세요. “my”라는 용어는 예제 코드에서는 사용할 수 있으나 실제 코드에서는 피해주세요.
2. 추가로 `dataSource.initialize()` 함수가 실행되고 나면, then()과 catch()를 사용해서 에러를 핸들링해줄 수 있습니다. 이렇게 하면, initialize() 함수가 실행되어서 database와 실제로 connection을 생성할 때, 성공했는지 실패했는지 여부를 확인할 수 있습니다.
3. typeorm 공식문서의 .initialize() 함수의 예제코드를 참고하여 수정해주세요!